.ci/jenkins/Jenkinsfile.post-release

```diff
@@ -98,6 +99,11 @@ String getKogitoVersion() {
     return params.KOGITO_VERSION
 }
 
+String getKogitoSWOperatorVersion() {
+    String[] kogitoVersionSplit = params.KOGITO_VERSION.split("\\.Final")
+    return "v.${kogitoVersionSplit[0]}"
```

Comment:
are you sure about the `v.` and not `v` ?

Reply:
it was a typo, I fixed it, thanks!